Task 1/Assets/Scripts/CuttingMeshCalculator.cs

```diff
@@ -0,0 +1,30 @@
+using UnityEngine;
+
+public class CuttingMeshCalculator
+{
+    public struct MeshesData
+    {
+        public Vector3 posStatic;
+        public Vector3 posDynamic;
+        public Vector3 sizeStatic;
+        public Vector3 sizeDynamic;
```

Comment:
Code style: public members in upper camel case 
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions